locales/base/translation.json

```
@@ -2312,5 +2312,21 @@
         "title": "More coming soon!"
       }
     }
+  },
+  "earnFlow": {
+    "addCrypto": {
```

nit: to be more explicit
```suggestion
    "addCryptoBottomSheet": {
```